src/backend/src/form-responses/form-responses.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { FormResponse } from 'src/entities/formResponse.entity';
+import { FormResponsesService } from './form-responses.service';
+import { CreateFormResponseDto, CreateMultiFormResponsesDto } from './dto';
+
+@Controller('form-responses')
+export class FormResponsesController {
+  constructor(private readonly formResponsesService: FormResponsesService) {}
+
+  @Post()
+  async create(
+    @Body('userId') userId: number,
+    @Body('questionId') questionId: number,
+    @Body('formResponse') formResponseData: CreateFormResponseDto,
+  ): Promise<FormResponse> {
```

Comment:
do we want to return the whole form response here? Or maybe we just return void. I can't really see a reason that we need to post back the content of the question. Just knowing it didn't fail should be enough

Reply:
Yeah that makes sense!